ADR/0037-integration-service-promotes-to-GCL-immediately.md

```diff
@@ -20,7 +24,8 @@ that was newly built is updated/overwritten to complete the Snapshot creation.
 The Global Candidate List for the newly built component would only be updated once all required integration tests
 for the created Snapshot have passed successfully.
 See more about this in [ADR 16. Integration Service Promotion Logic](0016-integration-service-promotion-logic.html) and
-[ADR 15. The Two-phase Architecture of the Integration Service](0015-integration-service-two-phase-architecture.html).
+[ADR 15. The Two-phase Architecture of the Integration Service](0015-integration-service-two-phase-architecture.html) and
+[ADR 38. Integration service removes composite snapshots and logic around them](0038-integration-service-composite-removal.html).
```

Comment:
I don't think you need to link to the ADR in the context, just in the superseded. The context is more for prior art/background as far as I understand it.

Reply:
You are correct, let me remove it.